Gemfile

```diff
@@ -30,10 +30,13 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 gem 'composite_primary_keys'
+gem 'nas-yahoo_stock'
```

Comment:
afaict, you don't actually use either of these gems?  You are making direct, hand rolled webservice calls to yql using net libraries, so you could probably remove both of these?